tests/test_datamaster.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+
+import pandas as pd
+
+from spacebench.datasets.datamaster import DataMaster
+
+
+class TestDataMaster(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
```

Comment:
You do not need to initialize the class. Please use `setUp` and `tearDown` functions. Here is an example:

```python
import unittest

class WidgetTestCase(unittest.TestCase):
    def setUp(self):
        self.widget = Widget('The widget')

    # Your tests go here

    def tearDown(self):
        self.widget.dispose()
```

Reply:
fixed